src/components/shared/__tests__/Rating.test.tsx

```diff
@@ -1,15 +1,18 @@
-import 'react-native';
-
 import * as React from 'react';
 
 import Rating from '../Rating';
 // Note: test renderer must be required after react-native.
+import { TouchableOpacity } from 'react-native';
 import renderer from 'react-test-renderer';
 
 let props: any;
-let component: React.ReactElement;
+
 // let testingLib: RenderResult;
 
+const component = (props?): React.ReactElement => {
```

Comment:
I think `props` should be follow [this interface rule](https://github.com/dooboolab/dooboo-ui-native/blob/f52dd3d302ab3135490681e99a3a8409ba0fd9ba/src/components/shared/Rating/index.tsx#L5)

`total` and `value` should be included.